cpp/formatting/README.md

```diff
@@ -9,6 +9,11 @@ code formatting conventions adopted by BlueBrain HPC team.
 
 # Conventions
 
+## Names and Order of Includes
```

Comment:
```suggestion
## Ordering of Includes
```
I didn't really see anything related to *naming* the includes in that link? Seems more about the sorting order… maybe I just missed it?